tasks/pg_utils/database.py

```diff
@@ -183,12 +232,16 @@ def read_db_connect_info(param_source):
     return dbconnect_info
 
 
-def get_db_connect_info(env_or_ssm, param_name, decrypt=False):
-    """
+def get_db_connect_info(env_or_ssm: str, param_name, decrypt: bool = False):
```

Comment:
Should there be a `-> <type>` in this function definition?

Reply:
Fixed